toolchest_client/query.py

```diff
@@ -185,51 +185,49 @@ def _upload(self, input_file_paths, input_prefix_mapping):
                 upload_response.raise_for_status()
             except HTTPError as e:
                 # todo: this isn't propagating as a failure
```

Comment:
Is this todo still true? I added it – and forgot to remove it – when testing STAR parallelization. I see that you added a `force_raise=True`.

Reply:
Update: looks like this was just due to a default value change.

Reply:
I'm not sure if there's still work to be done here. Could you share what inputs invoked the non-propagated failure?